### PR TITLE
Improved handling of CRYST1 records #3446

### PR DESCRIPTION
**Pull request by HrishiDhondge**

### CODE REVIEW

File: wrappers/python/openmm/app/internal/pdbstructure.py [hunk 1]
```diff
@@ -196,7 +196,8 @@ def _load(self, input_stream):
                 alpha = float(pdb_line[33:40])*math.pi/180.0
                 beta = float(pdb_line[40:47])*math.pi/180.0
                 gamma = float(pdb_line[47:54])*math.pi/180.0
-                self._periodic_box_vectors = computePeriodicBoxVectors(a_length, b_length, c_length, alpha, beta, gamma)
+                if not (a_length == b_length == c_length == 0):
```
Comment (swails) [inline wrappers/python/openmm/app/internal/pdbstructure.py:199]: What about if 1 or 2 dimensions are zero?  Simulations won't work well in that case, either.

Comment (swails) [reply wrappers/python/openmm/app/internal/pdbstructure.py:199]: Thanks for the correction. Edited my suggestion.

Comment (HrishiDhondge) [reply wrappers/python/openmm/app/internal/pdbstructure.py:199]: Yes, this suggestion seems more precise.

**Comment by peastman** [comment]
This fixes #3446.

**Comment by peastman** [comment]
Thanks!

**Pull request merged**